source/security/threatmonitoring/non-ukfast-install.md

```diff
@@ -0,0 +1,128 @@
+# Installing Threat Monitoring on a non-UKFast server
+
+```eval_rst
+.. warning::
+   This documentation is for setting on a non-UKFast hosted server with Threat Monitoring. Instructions on how to install Threat Monitoring and Response on a UKFast hosted server can be found in our [Installing Threat Monitoring on a UKFast hosted server](/security/threat-monitoring/ukfast-hosted-install) documentation.
```

Comment:
Check this about hyperlinks in `eval_rst` blocks: https://github.com/ukfast/docs.ukfast.co.uk#hyperlinks